processFlows.dscp

```diff
@@ -11,16 +11,25 @@
 token InitiatedCert {
   hydrogen_owner: Role,
   energy_owner: Role,
+  regulator: Role,
   hydrogen_quantity_mwh: Literal,
   commitment: Literal,
 }
 
 token IssuedCert {
   hydrogen_owner: Role,
   energy_owner: Role,
+  regulator: Role,
   embodied_co2: Literal,
 }
 
+token RevokedCert {
+  hydrogen_owner: Role,
+  energy_owner: Role,
+  regulator: Role,
+  reason: Literal,
```

Comment:
I also think this should be a `File`

Reply:
also change to be a File, to be fair also think it could be represented in more way than a plain text